changelogs/unreleased/4858-jxun

```diff
@@ -0,0 +1 @@
+Remove VolumeSnapshot created during backup, when the VolumeSnapshotClass's DeletionPolicy is set to Delete.
```

Comment:
Is this correct, I don't think it deletes the vs based on the DeletionPolicy.

Reply:
Thanks, forgot to update this.